blockmanager.go

```diff
@@ -1273,10 +1280,10 @@ func (b *blockManager) NewPeer(p *peer) {
 
 // QueueTx adds the passed transaction message and peer to the block handling
 // queue.
-func (b *blockManager) QueueTx(tx *btcutil.Tx, p *peer) {
+func (b *blockManager) QueueTx(tx *btcutil.Tx, p *peer.Peer) {
 	// Don't accept more transactions if we're shutting down.
 	if atomic.LoadInt32(&b.shutdown) != 0 {
-		p.txProcessed <- struct{}{}
+		p.TxProcessed <- struct{}{}
```

Comment:
I believe we're going to want a better mechanism for this.  It is mixing peer and block manager state and won't fit in well with other callers.